### PR TITLE
PBM-1011 Release notes 2.0.3

### PR DESCRIPTION
**Pull request by nastena1606**

### CODE REVIEW

File: docs/release-notes/2.0.3.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+* [PBM-998](https://jira.percona.com/browse/PBM-998), [PBM-1002](https://jira.percona.com/browse/PBM-1002) - Ensure that PBM waits for the `mongod` to stop / start completely during physical restore
+* [PBM-1009](https://jira.percona.com/browse/PBM-1009) - Add AWS IRSA credentials support (Thanks to the user pastequo for contributing to this feature)
+ 
+
+## Bugs Fixed
+
+* [PBM-932](https://jira.percona.com/browse/PBM-932) - Fix the data access from `mongos` after selective restore on a new cluster
+* [PBM-981](https://jira.percona.com/browse/PBM-981) - Prevent showing the ERROR status while backup is running
+* [PBM-1006](https://jira.percona.com/browse/PBM-1006) - Fix the `pbm-agent` crash during physical restore by initializing the logger for storage
```
Comment (sandraromanchenko) [inline docs/release-notes/2.0.3.md:32]: please also add [PBM-1003] (https://jira.percona.com/browse/PBM-1003)

**Review by sandraromanchenko — CHANGES_REQUESTED** [review]

File: docs/release-notes/2.0.3.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+ 
+
+## Bugs Fixed
+
+* [PBM-932](https://jira.percona.com/browse/PBM-932) - Fix the data access from `mongos` after selective restore on a new cluster
+* [PBM-981](https://jira.percona.com/browse/PBM-981) - Prevent showing the ERROR status while backup is running
+* [PBM-1003](https://jira.percona.com/browse/PBM-1003) - Fix the ability to restore with remapping replica set names (Thanks to user Piotr Krogulec for reporting this issue and to user ziollek for contributing to it)
+* [PBM-1006](https://jira.percona.com/browse/PBM-1006) - Fix the `pbm-agent` crash during physical restore by initializing the logger for storage 
+* [PBM-1008](https://jira.percona.com/browse/PBM-1008) - Fix the selective PITR restore for a collection created after the full backup
+* [PBM-1012](https://jira.percona.com/browse/PBM-1012) - Collection is not dropped during selective oplog replay on create command
```
Comment (sandraromanchenko) [inline docs/release-notes/2.0.3.md:35]: remove this one please, it's decided to move it to the next version

**Review by sandraromanchenko — CHANGES_REQUESTED** [review]

File: docs/release-notes/2.0.3.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+| Release date | January 11, 2023  |
+|------------- | ---------------|
+| Installation | [Installing Percona Backup for MongoDB](../installation.md) |
+
+
+Percona Backup for MongoDB is a distributed, low-impact solution for consistent backups of MongoDB sharded clusters and replica sets. This is a tool for creating consistent backups across a MongoDB sharded cluster (or a non-sharded replica set), and for restoring those backups to a specific point in time.
+
+## Release Highlights
+
+* [Incremental physical backups](usage/incremental-backup.md) enable you to reduce storage costs and facilitate data safety for business crucial data. By saving only the differences results in faster completion time and makes IB much smaller in size compared to full backups. As such, you save on storage space and data transfer in case of cloud setups. This is the [technical preview feature](reference/glossary.md#technical-preview-feature), yet we welcome your feedback to improve the functionality.
```
Comment (janwieremjewicz) [inline docs/release-notes/2.0.3.md:12]: Did we introduce the abbreviation IB?

**Review by janwieremjewicz — COMMENTED** [review]
Not sure about the IB abbreviation. Other than this is ok

File: docs/release-notes/2.0.3.md (hunk 3 above)
Comment (nastena1606) [reply docs/release-notes/2.0.3.md:12]: No, we didn't

File: docs/release-notes/2.0.3.md (hunk 2 above)
Comment (nastena1606) [reply docs/release-notes/2.0.3.md]: Fixed

**Review by sandraromanchenko — APPROVED** [review]

**Review by dAdAbird — APPROVED** [review]

**Pull request merged**